lib/feature/auth/forgot_password/view/ui/forgot_password_page_3.dart

```diff
@@ -0,0 +1,249 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import 'confirm_new_password_page.dart';
+
+import '../../../../../utils/widgets/alta_svg.dart';
+import '../../../../../utils/widgets/alta_scaffold.dart';
+import '../../../../../utils/widgets/alta_icon_button.dart';
+
+import '../../../../../utils/widgets/alta_text_button.dart';
+import '../../../../../utils/widgets/alta_logo.dart';
+import '../../../../../utils/constant/alta_border_radius.dart';
+import '../../../../../utils/widgets/alta_text.dart';
+import '../../../../../utils/constant/alta_color.dart';
+import '../../../../../utils/constant/alta_spacing.dart';
+import '../../../../../utils/widgets/alta_primary_button.dart';
+
+class ForgotPasswordPage3 extends StatelessWidget {
+  const ForgotPasswordPage3({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final Uri url = Uri.parse(
+        'https://flexiprogram-api.alterraacademy.id/api/v1/auth/email-verification/eyJpdiI6IjYxSnJKTWRHYzd1ckFEcHo1Y2VQV3c9PSIsInZhbHVlIjoiT3RKd0RSVWxqdU1GWjhHUCtGTlwvcXU4TnlvOHZCKzFOUElqaVk0NnlRTlZQWjhpS3RKRVNcLzFRK3JHWE41eVwvaCIsIm1hYyI6ImFmYmY4OGE0NmNmZDQ0N2JjM2M3OWE5NmY2NTEzYTMwMjg1YjRlNjQ3YzY5NGE2YzI3OTA5NWQyYjJkYmIwOGUifQ==');
```

Comment:
create constant class to store this link, 

AltaURL{
const urlXXX = "https://flexiprogram-api.alterraacademy.id/api/v1/auth/email-verification/eyJpdiI6IjYxSnJKTWRHYzd1ckFEcHo1Y2VQV3c9PSIsInZhbHVlIjoiT3RKd0RSVWxqdU1GWjhHUCtGTlwvcXU4TnlvOHZCKzFOUElqaVk0NnlRTlZQWjhpS3RKRVNcLzFRK3JHWE41eVwvaCIsIm1hYyI6ImFmYmY4OGE0NmNmZDQ0N2JjM2M3OWE5NmY2NTEzYTMwMjg1YjRlNjQ3YzY5NGE2YzI3OTA5NWQyYjJkYmIwOGUifQ==";
}